submissions/chimoney-redeem-airtime/src/components/Home.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components';
+import {CHI_REF} from '../SecretKeys';

```

Comment:
Is this chiRef coming from a local file?

I am curious about this page because my original assumption for the home page is that is should potentially ask them to paste a ChiRef or a RedeemLink from Chimoney in a form field and then use the Chiref to redirect to the redeem airtime page

Reply:
Yes it is..
was kind of confused about how to go about getting the chiRef in the first place. 
but seems your idea is better.
Let me quickly modify the project to that effect. 🏃‍♀️